doc/source/visualization.rst

```diff
@@ -1049,6 +1049,46 @@ be colored differently.
 Plot Formatting
 ---------------
 
+Setting the plot style
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+From version 1.5 and up, matplotlib offers a range of preconfigured plotting style. Setting these
+these styles can be used to easily give the plots have the general look that you want.
+Setting the can style is as easy as calling ``matplotlib.style.use(my_plot_style)`` before
+creating your plot.
+
```

Comment:
I don't think the examples of plot style are necessary here, again just point to the mpl docs.